xc7/utils/prjxray_physical_tile_import.py

```diff
@@ -0,0 +1,250 @@
+#!/usr/bin/env python3
+"""
+Import the top level tile information from Project X-Ray database
+files.
+
+Project X-Ray specifies the connections between tiles and the connect
+between tiles and their sites.  prjxray_tile_type_import generates a VPR tile
+that has the correct tile pin location, switchblock location and fc information.
+
+Optionally, if there are available equivalent tiles, the tile information is filled
+with the equivalent tile pin mapping as well. This is needed by VPR to have a correct
+translation of the pin names when using the equivalent tile.
+
+"""
+
+from __future__ import print_function
+import argparse
+import os
+import sys
+import prjxray.db
+import os.path
+import simplejson as json
+
+import lxml.etree as ET
+
+XI_URL = "http://www.w3.org/2001/XInclude"
+
+# Macros used to select relevant port names
+PORT_TAGS = ['input', 'output', 'clock']
+
+
+def prefix_name(tile):
```

Comment:
Rather than c/p, can you import these functions from prjxray_tile_import?

Reply:
Done